### PR TITLE
fix(tools/respecDocWriter): fix regressions

### PR DESCRIPTION
**Pull request by sidvishnoi**

### CODE REVIEW

**Comment by sidvishnoi** [comment]
Hmmm. We can actually remove that haltFlags argument altogether.

File: tools/respecDocWriter.js [hunk 1]
```diff
@@ -174,7 +174,7 @@ async function getVersion(page) {
 async function checkIfReSpec(page) {
   const isRespecDoc = await page.evaluate(isRespec);
   if (!isRespecDoc) {
-    const msg = `🕵️‍♀️  That doesn't seem to be a ReSpec document. Please check manually: ${page.url}`;
+    const msg = `🕵️‍♀️  That doesn't seem to be a ReSpec document. Please check manually: ${page.url()}`;
```
Comment (sidvishnoi) [inline tools/respecDocWriter.js:177]: We had this since forever :scream:

File: tools/respecDocWriter.js [hunk 2]
```diff
@@ -292,17 +289,13 @@ function handleConsoleMessages(page, haltFlags, onError, onWarning) {
...
       case "error":
-        onError({ message: text });
-        haltFlags.error = true;
-        break;
+        return onError({ message: text });
       case "warning":
         // Ignore polling of respecDone
         if (/document\.respecDone/.test(text)) {
           return;
         }
```
Comment (sidvishnoi) [inline tools/respecDocWriter.js:297]: I think we can remove this now? Or is it needed in some really old ReSpec?
```suggestion
```

File: tools/respecDocWriter.js (hunk 1 above)
Comment (marcoscaceres) [reply tools/respecDocWriter.js:177]: oh! that explains things.

File: tools/respecDocWriter.js (hunk 2 above)
Comment (marcoscaceres) [reply tools/respecDocWriter.js:297]: Yeah, we can remove.

**Review by marcoscaceres — APPROVED** [review]

**Comment by sidvishnoi** [comment]
I'm going to split this into 2 PRs (fix, then remove unused code)

**Pull request merged**